Monika After Story/game/script-topics.rpy

```diff
@@ -4412,6 +4411,44 @@ label monika_pets:
     m 1a "Until I find a way out, could you promise to keep me from feeling lonely?"
     m 5a "I'll see if I can get that plush one in here! Oh- don't worry, you're still my favorite~"
     return
+    
+    
+    init 5 python:
+    # List of keywords for the topic.
+    for key in ['dog','dogs','puppy','puppies']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('dog_discussion') # Identifier
+
+label dog_discussion:
```

Comment:
We like to keep our label names consistent. All monika topics have a label like `monika_topicname`. Please change your topic's label name so it follows our standards.